plugins/device/cmd/nvidia/README.md

```diff
@@ -0,0 +1,23 @@
+This package provides an implementation of nvidia device plugin
+
+# Behavior
+
+Nvidia device plugin uses NVML bindings to get data regarding available nvidia devices and will expose them via Fingerprint RPC. Device would not be visible by nomad if it is listed in `blacklisted_gpu_ids` config section. Plugin sends statistics for every fingerprinted device with `stats_period` period.
+
```

Comment:
change "Plugin sends statistics for every fingerprinted device with `stats_period` period." to "Plugin sends statistics for fingerprinted devices every `stats_period` period."